src/server/index.js

```diff
@@ -49,7 +46,10 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, '../..', 'public')));
 
 app.use('/', require('./routes/'));
-defineSockets(io);
+app.use('/api', require('./routes/rest'));
+require('./io').set(server);
+require('./routes/sockets');
```

Comment:
@maxkfranz I made the changes to the best of my understanding, but this still feels like something isn't quite right. I needed to add the second require statement so that the sockets were actually defined in the io object.